plugins/ccgx-dmc/fu-ccgx-dmc-device.c

```diff
@@ -738,6 +738,20 @@ fu_ccgx_dmc_device_ensure_factory_version(FuCcgxDmcDevice *self)
 	}
 }
 
+static gboolean
+fu_ccgx_dmc_device_probe(FuDevice *device, GError **error)
+{
+	FuCcgxDmcDevice *self = FU_CCGX_DMC_DEVICE(device);
+	g_autoptr(FuUsbInterface) intf = NULL;
+
+	/* find the correct vendor-specific interface */
+	intf = fu_usb_device_get_interface(FU_USB_DEVICE(self), 0xFF, 0x03, 0x00, error);
+	if (intf == NULL)
+		return FALSE;
```

Comment:
Won't this cause probe failure on other devices? Or they all have this one?

Reply:
My device has that interface too.